sql/src/main/java/io/crate/execution/engine/aggregation/impl/MaximumAggregation.java

```diff
@@ -45,15 +45,27 @@
     private final FunctionInfo info;
 
     public static void register(AggregationImplModule mod) {
-        for (final DataType dataType : DataTypes.PRIMITIVE_TYPES) {
-            FunctionInfo functionInfo = new FunctionInfo(
-                new FunctionIdent(NAME, ImmutableList.of(dataType)), dataType, FunctionInfo.Type.AGGREGATE);
-
-            if (dataType instanceof FixedWidthType) {
-                mod.register(new FixedMaximumAggregation(functionInfo));
-            } else {
-                mod.register(new VariableMaximumAggregation(functionInfo));
-            }
+        for (var supportedType : DataTypes.PRIMITIVE_TYPES) {
```

Comment:
This test fails when applying the max function on the column that does not exist because max(undefined) is not recognized. I don't think that we can register the undefined type because all passed types will match the signature.
   https://github.com/crate/crate/blob/08d1315068432e52e7c9fcc531b116dda3628500/sql/src/test/java/io/crate/integrationtests/GroupByAggregateTest.java#L703

Shouldn't this test faill with unknow column exception?